lib/awful/placement.lua

```diff
@@ -1493,7 +1497,11 @@ function placement.next_to(d, args)
         if fit then break end
     end
 
-    local ngeo, dir = unpack(does_fit[pref_name] or {}) --FIXME why does this happen
+    if not pref_name then return end
+
+    assert(does_fit[pref_name])
+
+    local ngeo, dir = does_fit[pref_name]
```

Comment:
I don't think you wanted to actually remove the `unpack` here.

Reply:
Yeah, sorry for the noise.